src/fut/mod.rs

```diff
@@ -44,7 +44,38 @@ use actor::Actor;
 /// Trait for types which are a placeholder of a value that may become
 /// available at some later point in time.
 ///
-/// This is similar to `futures::Future` trait, except it works with `Actor`
+/// `ActorFuture` is very similar to a regular `Future`, except for the fact that `map`ing over it
+/// works with closures accepting the relevant actor and its context.
+///
+/// `ActorFuture` is required in cases where future processing requires access to the actor ot its context -
+/// as this cannot be done by capturing a reference as a part of the closure.
+///
+/// Here is an example of a handler on a single actor, deferring work to a different actor, and
+/// then updating the initiating actor's state:
+///
+/// ```rust,ignore
+/// impl Message for SomeMessage {
+///     type Result = ();
+/// }
+///
+/// impl Handler<DeferredWork> for OriginalActor {
+///     type Result = ResponseActFuture<Self, (), Error>;
+///
+///     fn handle(&mut self, _msg: Refresh, ctx: &mut Context<Self>) -> Self::Result {
+///         let send_to_other = self.other_actor_addr
+///             .send(OtherMessage::new())
+///             .map_err(Error::from);
+///
+///         let update_self =
+///             wrap_future::<_, Self>(send_to_other).map(|result, actor, _ctx| {
```

Comment:
Add note that there is also [into_actor](https://docs.rs/actix/0.7.3/actix/fut/trait.WrapFuture.html#tymethod.into_actor)